tests/pipelines/stable_diffusion_2/test_stable_diffusion_v_pred.py

```diff
@@ -242,9 +242,9 @@ def test_stable_diffusion_v_pred_fp16(self):
         assert image.shape == (1, 64, 64, 3)
 
 
-@slow
```

Comment:
Stable Diffusion 2 is an important pipeline - we should try to leave this as a slow test. Can we instead try to speed it up, e.g. by running only 10 inference steps